tests/torch/quantization/test_functions.py

```diff
@@ -368,13 +503,16 @@ def test_quantize_asymmetric_backward(self, _seed, input_size, bits, use_cuda, i
             test_value.sum().backward()
```

Comment:
This place is producing `inf` values when `is_fp16 == True`
@alexsu52

Reply:
Stands to reason. Probably will be fixed with your accumulator PR

Reply:
Only thing we can do is to not use `.sum()` here, because `test_value` should be in FP16 precision

Reply:
Could you generate and fix valid input values which do not produce inf?